build.psm1

```diff
@@ -681,25 +681,29 @@ function Publish-PSTestTools {
 
     Find-Dotnet
 
-    $tools = @("$PSScriptRoot/test/tools/EchoArgs", "echoargs"), @("$PSScriptRoot/test/tools/CreateChildProcess", "createchildprocess")
+    $tools = ,@("${PSScriptRoot}/test/tools/TestExe", "testexe")
     if ($Options -eq $null)
```

Comment:
That works too, but `[pscustomobject]@{Path="${PSScriptRoot}/test/tools/TestExe";Output="testexe"}` expresses the intent more clearly to me.

---
In reply to: [121528687](https://github.com/PowerShell/PowerShell/pull/3982#discussion_r121528687) [](ancestors = 121528687)